src/CommunityToolkit.Maui/Alerts/Snackbar/SnackBar.shared.cs

```diff
@@ -136,4 +143,28 @@ void OnDismissed()
 		IsShown = false;
 		_weakEventManager.HandleEvent(this, EventArgs.Empty, nameof(Dismissed));
 	}
+}
+
+/// <summary>
+/// Extension methods for <see cref="VisualElement"/>.
+/// </summary>
+public static class VisualElementExtension
+{
+	/// <summary>
+	/// Display snackbar with the anchor
+	/// </summary>
+	/// <param name="visualElement">Anchor element</param>
+	/// <param name="message">Text of the snackbar</param>
+	/// <param name="actionButtonText">Text of the snackbar button</param>
+	/// <param name="action">Action of the snackbar button</param>
+	/// <param name="duration">Snackbar duration</param>
+	/// <param name="visualOptions">Snackbar visual options</param>
+	/// <returns><see cref="Snackbar"/></returns>
+	public static Task DisplaySnackbar(
```

Comment:
it is useful to return snackbar instance here. we can later dismiss it, subscribe to events, etc.

Reply:
No, let's continue to treat `DisplaySnackbar` like `DisplayAlert`.

If a user wants to do more with a Snackbar, they can use `new Snackbar()`.